crypto/fipsmodule/digest/digest.c

```diff
@@ -114,6 +117,9 @@ int EVP_MD_CTX_cleanup(EVP_MD_CTX *ctx) {
 }
 
 void EVP_MD_CTX_cleanse(EVP_MD_CTX *ctx) {
+  if (ctx == NULL || ctx->md_data == NULL || ctx->digest == NULL) {
+    return;
+  }
```

Comment:
NP: Normally all `ctx->digest` should have a `ctx_size`, but would it be worth considering adding `assert(ctx->digest->ctx_size != NULL && sizeof(ctx->md_data) == ctx->digest->ctx_size)` if we're adding new checks?